experiment_config/bidaf.json

```diff
@@ -1,6 +1,10 @@
 {
   "dataset_reader": {
     "type": "squad",
+    "tokenizer": {
```

Comment:
Yes, I'm intending to clean up the word splitter stuff (it doesn't currently use `Registrable`, e.g.), and change the default to spacy.